src/common/filesys.cpp

```diff
@@ -130,7 +130,9 @@ wxString wxFileSystemHandler::GetMimeTypeFromExt(const wxString& location)
 
         return mime;
     }
+#if wxUSE_SYSTEM_OPTIONS
     else
+#endif
```

Comment:
I might be misreading this, but couldn't we just get rid of this `else` completely? The `if` branch always returns, so I think we could remove the `else` and simplify the code instead of complicating it even further.

Reply:
That is true indeed, after a closer look. I've adjusted the PR to just remove the `else`.